contrib/rtlbrowse/logfile.c

```diff
@@ -93,1390 +89,1308 @@ static GtkTextTag *blue_tag = NULL, *fwht_tag = NULL;
-					}
-				}
-
-			gtk_text_iter_backward_char(&end);
-			for(;;)
-				{
-				gtk_text_iter_forward_char(&end);
-		               	sel2 = gtk_text_buffer_get_text(gtk_text_view_get_buffer(GTK_TEXT_VIEW(text)),
-                                              	&start, &end, FALSE);
-				if(!sel2) break;
-				ch = *(sel2 + strlen(sel2) - 1);
-				g_free(sel2);
-				if(!is_identifier(ch))
-					{
-					gtk_text_iter_backward_char(&end);
-					break;
-					}
-				}
-
-		 	sel2 = gtk_text_buffer_get_text(gtk_text_view_get_buffer(GTK_TEXT_VIEW(text)),
-						&start, &end, FALSE);
-
-			g_free(sel);
-			sel = sel2;
-			sel2 = NULL;
-			}
-
-		/* gtk_text_buffer_delete_selection (gtk_text_view_get_buffer(GTK_TEXT_VIEW(text)), 0, 0); ...no need to delete because of update_ctx_when_idle() */
-		}
-
-	}
-
-
-if(sel)
-	{
-	JRB strs, node;
-	int fd;
-	char *fname = tmpnam_rtlbrowse("rtlbrowse", &fd);
-	FILE *handle = fopen(fname, "wb");
-	int lx;
-	int degate = 0;
-	int cnt = 0;
-
-	if(!handle)
-		{
-        	fprintf(stderr, "Could not open cutpaste file '%s'\n", fname);
-        	return;
-        	}
-	fprintf(handle, "%s", sel);
-	fclose(handle);
-	if(fd>=0) close(fd);
-
-	v_preproc_name = fname;
-	strs = make_jrb();
-    	while((lx = yylex()))
-        	{
-        	char *pnt = yytext;
-
-		if(!verilog_2005)
-			{
-			if(lx == V_KW_2005) lx = V_ID;
-			}
-
-                if(lx==V_ID)
-                        {
-			if(!degate)
-				{
-				JRB str = jrb_find_str(strs, pnt);
-				Jval jv;
-				jv.v = NULL;
-				if(!str)
-					{
-					jrb_insert_str(strs, strdup(pnt), jv);
-					cnt++;
-					}
-				}
-                        }
-		else if(lx==V_IGNORE)
-			{
-			if(*pnt == '[') degate = 1;
-			else if(*pnt == ']') degate = 0;
-			}
-		}
-
-	unlink(fname);
-	free(fname);
-
-	if(cnt)
-		{
-		int title_len = 5 + strlen(ctx->title) + 1;
-		char *tpnt = calloc(1, title_len + 1);
-		char *op = ctx->title;
-		char *np = tpnt;
-		char *mlist = NULL;
-		int mlen = 0;
-
-		strcpy(np, "{net ");
-		np+=5;
-		while(*op)
-			{
-			if(*op == '.') *np = ' '; else *np = *op;
-			op++;
-			np++;
-			}
-		*np = ' ';
-
-		jrb_traverse(node, strs)
-			{
-			int slen = strlen(node->key.s);
-			int singlen = title_len + slen + 2;
-			char *singlist = calloc(1, singlen + 1);
-			memcpy(singlist, tpnt, title_len);
-			strcpy(singlist + title_len, node->key.s);
-			strcpy(singlist + title_len + slen, "} ");
-
-			if(mlist)
-				{
-				mlist = realloc(mlist, mlen + singlen + 1);
-				strcpy(mlist + mlen, singlist);
-				mlen += singlen;
-				}
-				else
-				{
-				mlist = strdup(singlist);
-				mlen = singlen;
-				}
-
-			free(singlist);
-			free(node->key.s);
-			}
-
-		gtk_selection_data_set(selection_data,GDK_SELECTION_TYPE_STRING, 8, (guchar*)mlist, mlen);
-		free(mlist);
-
-		update_ctx_when_idle(text);
-
-		free(tpnt);
-		}
-		else
-		{
-		update_ctx_when_idle(text);
-		}
-
-	jrb_free_tree(strs);
-	g_free(sel);
-	}
-
-}
-
-
-
+#define WAVE_DRAG_TAR_NAME_0 "text/plain"
```

Comment:
Let's just remove the drag and drop code entirely, we can always refer back to the GIT history or LTS version if we need to.